.github/workflows/sycl-linux-build.yml

```diff
@@ -50,9 +50,6 @@ on:
         description: 'Artifacts retention period'
         type: string
         default: 3
-      ref:
```

Comment:
Note for reviewers: I added these changes (and one below) previously, but they are not needed actually, so removing then.